WORKFLOW.md

```diff
@@ -0,0 +1,69 @@
+# Grafana workflow
+
+This document is based on [GOVERNANCE.md](GOVERNANCE.md). We assume good faith and intend to keep all processes as lightweight as possible but as specific as required. In case of disagreements about anything in this document, GOVERNANCE.md applies.
+
+The key words “MUST”, “MUST NOT”, “REQUIRED”, “SHALL”, “SHALL NOT”, “SHOULD”, “SHOULD NOT”, “RECOMMENDED”, “MAY”, and “OPTIONAL” in this document are to be interpreted as described in [RFC2119](http://tools.ietf.org/html/rfc2119).
+
+Git and [GitHub terminology](https://help.github.com/en/github/getting-started-with-github/github-glossary) are used throughout this document.
+
+Team members and their access to repositories is maintained through [GitHub teams](https://help.github.com/en/github/setting-up-and-managing-organizations-and-teams/about-teams). Team maintainers add and remove team members as per GOVERNANCE.md
+
+
+# Code changes
+
+## Proposing changes
+
+Examples of proposed changes are overarching architecture, subsets, and specific code/graphical elements. Proposed changes SHOULD cover the big picture and intention, but individual parts SHOULD be split into the smallest possible changes. Changes SHOULD be based on and target the master branch. Depending on size of the proposed change, they SHOULD be discussed, in increasing order of change size and complexity:
+Directly in a Pull Request - this MAY be done, but SHOULD not be the common case
+Issue
+Developer mailing list
+Design document, shared via Google Docs, accessible to at least all team members
+
+Significant changes MUST be discussed and agreed upon with the relevant subsystem maintainers.
+
+## Merging Pull Requests
+Depending on the size and complexity of a Pull Request, different requirements MUST be applied. Any team member contributing substantially to a Pull Request MUST NOT count against review requirements.
```

Comment:
```suggestion

Depending on the size and complexity of a PR, different requirements MUST be applied. Any team member contributing substantially to a PR MUST NOT count against review requirements.
```

Reply:
Fix the content in lines 26-33 so that they make a coherent paragraph or two. Also, end the sentences with periods.